### DISCUSSION TITLE
Retrospective discussion about the recent rejections of JSON Schema in GSoC and GSoD

### DISCUSSION THREAD

**Discussion by benjagm** [json-schema-org/community#369] [General]
# Retrospective discussion about the recent rejections of JSON Schema in GSoC and GSoD

This 2023, JSON Schema org applied as a mentoring organization [GSoC](https://summerofcode.withgoogle.com/) and [GSoD](https://developers.google.com/season-of-docs), and our applications were both rejected. We knew we had prepared our GSoC application in just a few weeks; hence our expectations were low. However, we were confident that with our use case for GSoD this is this rejection was a surprise.

This discussion aims to run a Community retrospective to identify what worked and did work to identify improvements to get better results in future applications.

Find our application below:
* **GSoC** Application [here](https://github.com/json-schema-org/community/blob/main/gsoc/gsoc-2023.md)
* **GSoD** Application [here](https://github.com/orgs/json-schema-org/discussions/359)

Please leave your comments for any of these options:
- What worked
- What did not work
- What aspects do we need to improve for next year

Once we have enough feedback we'll turn the insights into actions and a plan.

**Comment by benjagm** [comment]
This is my personal analysis. 

What worked: 
- It is great that step up and applied for the 1st time.
- Even though we were rejected to GSoC we managed to include 2 ideas as part of Postman's application and the contributor's response has been amazing. The positive impact in the community is evident.
- We have quality mentors and we gathered great feedback from contributors. We contributors to Bowtie increased considerably.
- Though applying we are learning improvement areas.

What did not worked:
- Next time we need months of planing, promotion and discussions instead of days. There was no community engagement around our applications.
- Early involvement of TSC, Mentors and the Community.
- Our current contributors experience/strategy needs lots of improvement for both code and docs.  We need to make that strategy explicit and part of the community life.

What aspects do we need to improve for next year:
- To apply to GSoD we need 1st to formalize first our vision for JSON Schema docs so that the application is focused on materialize that vision with a strong use case.
- To apply to CSoC we need to improve our contributors experience. We need more projects to contribute to, more incentives, more visibility for the contributors group, and a solid plan to engage with implementers outside the github organisation.

**Reply by segun-codes** [reply]
Hi @benjagm, great work there. Do you want to publish or provide link to the feedback from GSoC upon rejection of the applications. I believe this will offer further background on the circumstances that informed the rejection. Thanks!

**Reply by Relequestual** [reply]
@segun-codes unfortunatley, they do not seem to provide any feedback on the submission. This is sad.

**Reply by segun-codes** [reply]
@Relequestual, okay. Sad, indeed.
